ivle-sync.py

```diff
@@ -97,6 +116,10 @@ def get_token(self):
         if len(r.text) > 1000:  # hacky way to check if return is a HTML page
             return ''
 
+        credentials['TOKEN'] = r.text
+        if yes_no():
```

Comment:
Sorry for the miscommunication, I feel we should only prompt them if we're saving credentials (as they are quite sensitive). For tokens, it is fine to leave the user logged in.